pyOCD/coresight/ap.py

```diff
@@ -337,8 +337,10 @@ def _handle_error(self, error, num):
 
 class AHB_AP(MEM_AP):
     def init(self, bus_accessible=True):
-        super(AHB_AP, self).init(bus_accessible)
+        # Init super first with bus access disabled, so it doesn't read the ROM table.
+        super(AHB_AP, self).init(False)
```

Comment:
Reading the ROM table is done in `AccessPort.init()` if the `bus_accessible` parameter is True. `init()` keeps track of what is has successfully inited so you can call it multiple times. Register 0xf8 of APs is the ROM table address, and indicates a ROM table is available if it is not 0xffff_ffff and bit 0 is set.

Anyway, I'm renaming the IDR lookup table to `MEM_AP_IDR_TO_WRAP_SIZE` and moving setting of `auto_increment_page_size` to `MEM_AP.init()`.

The MDM-AP doesn't have a ROM table, btw.